From 343b3bd79d3a90262192a2ccb4ae3a1b8baf1168 Mon Sep 17 00:00:00 2001
From: Ethan Everett <eeverett@meraki.com>
Date: Mon, 19 Nov 2018 15:42:11 -0800
Subject: [PATCH 07/15] excluded dictionaries

Based on Jignesh's 006-excludeddictionaries patch for v2.2.5

	"Excluding dictionaries that do not seem to be necessary.
    I did not do a very thorough investigation of which ones
    are needed. This can be fine tuned later."
---
 share/dictionary | 328 +++++++++++++++++++++++------------------------
 1 file changed, 164 insertions(+), 164 deletions(-)

diff --git a/share/dictionary b/share/dictionary
index 39d8a65caf..865c404ab7 100644
--- a/share/dictionary
+++ b/share/dictionary
@@ -73,8 +73,8 @@ $INCLUDE dictionary.compat
 #
 #	i.e. don't do this.  Don't use these attributes
 #
-$INCLUDE dictionary.usr.illegal
-$INCLUDE dictionary.ascend.illegal
+#$INCLUDE dictionary.usr.illegal
+#$INCLUDE dictionary.ascend.illegal
 
 #
 #	IETF allocated attributes and values.  Split out into
@@ -92,24 +92,24 @@ $INCLUDE dictionary.rfc2869
 $INCLUDE dictionary.rfc3162
 $INCLUDE dictionary.rfc3576
 $INCLUDE dictionary.rfc3580
-$INCLUDE dictionary.rfc4072
+#$INCLUDE dictionary.rfc4072
 $INCLUDE dictionary.rfc4372
-$INCLUDE dictionary.rfc4603
+#$INCLUDE dictionary.rfc4603
 $INCLUDE dictionary.rfc4675
 $INCLUDE dictionary.rfc4679
-$INCLUDE dictionary.rfc4818
-$INCLUDE dictionary.rfc4849
-$INCLUDE dictionary.rfc5176
-$INCLUDE dictionary.rfc5447
-$INCLUDE dictionary.rfc5580
-$INCLUDE dictionary.rfc5607
-$INCLUDE dictionary.rfc5904
-$INCLUDE dictionary.rfc6519
-$INCLUDE dictionary.rfc6572
-$INCLUDE dictionary.rfc6677
-$INCLUDE dictionary.rfc6911
+#$INCLUDE dictionary.rfc4818
+#$INCLUDE dictionary.rfc4849
+#$INCLUDE dictionary.rfc5176
+#$INCLUDE dictionary.rfc5447
+#$INCLUDE dictionary.rfc5580
+#$INCLUDE dictionary.rfc5607
+#$INCLUDE dictionary.rfc5904
+#$INCLUDE dictionary.rfc6519
+#$INCLUDE dictionary.rfc6572
+#$INCLUDE dictionary.rfc6677
+#$INCLUDE dictionary.rfc6911
 $INCLUDE dictionary.rfc6929
-$INCLUDE dictionary.rfc6930
+#$INCLUDE dictionary.rfc6930
 $INCLUDE dictionary.rfc7055
 $INCLUDE dictionary.rfc7155
 $INCLUDE dictionary.rfc7268
@@ -125,39 +125,39 @@ $INCLUDE dictionary.iana
 #
 #	Vendor dictionaries are listed after the standard ones.
 #
-$INCLUDE dictionary.3com
-$INCLUDE dictionary.3gpp
-$INCLUDE dictionary.3gpp2
-$INCLUDE dictionary.acc
-$INCLUDE dictionary.acme
-$INCLUDE dictionary.actelis
-$INCLUDE dictionary.adtran
-$INCLUDE dictionary.airespace
-$INCLUDE dictionary.alcatel
-$INCLUDE dictionary.alcatel.sr
-$INCLUDE dictionary.alcatel-lucent.aaa
-$INCLUDE dictionary.alteon
-$INCLUDE dictionary.alvarion
-$INCLUDE dictionary.apc
-$INCLUDE dictionary.aptilo
-$INCLUDE dictionary.arbor
-$INCLUDE dictionary.arista
-$INCLUDE dictionary.aruba
-$INCLUDE dictionary.azaire
-$INCLUDE dictionary.ascend
-$INCLUDE dictionary.bay
-$INCLUDE dictionary.bintec
-$INCLUDE dictionary.bluecoat
-$INCLUDE dictionary.boingo
-$INCLUDE dictionary.broadsoft
-$INCLUDE dictionary.brocade
-$INCLUDE dictionary.bskyb
-$INCLUDE dictionary.bt
-$INCLUDE dictionary.cablelabs
-$INCLUDE dictionary.cabletron
-$INCLUDE dictionary.camiant
-$INCLUDE dictionary.checkpoint
-$INCLUDE dictionary.chillispot
+#$INCLUDE dictionary.3com
+#$INCLUDE dictionary.3gpp
+#$INCLUDE dictionary.3gpp2
+#$INCLUDE dictionary.acc
+#$INCLUDE dictionary.acme
+#$INCLUDE dictionary.actelis
+#$INCLUDE dictionary.adtran
+#$INCLUDE dictionary.airespace
+#$INCLUDE dictionary.alcatel
+#$INCLUDE dictionary.alcatel.sr
+#$INCLUDE dictionary.alcatel-lucent.aaa
+#$INCLUDE dictionary.alteon
+#$INCLUDE dictionary.alvarion
+#$INCLUDE dictionary.apc
+#$INCLUDE dictionary.aptilo
+#$INCLUDE dictionary.arbor
+#$INCLUDE dictionary.arista
+#$INCLUDE dictionary.aruba
+#$INCLUDE dictionary.azaire
+#$INCLUDE dictionary.ascend
+#$INCLUDE dictionary.bay
+#$INCLUDE dictionary.bintec
+#$INCLUDE dictionary.bluecoat
+#$INCLUDE dictionary.boingo
+#$INCLUDE dictionary.broadsoft
+#$INCLUDE dictionary.brocade
+#$INCLUDE dictionary.bskyb
+#$INCLUDE dictionary.bt
+#$INCLUDE dictionary.cablelabs
+#$INCLUDE dictionary.cabletron
+#$INCLUDE dictionary.camiant
+#$INCLUDE dictionary.checkpoint
+#$INCLUDE dictionary.chillispot
 $INCLUDE dictionary.cisco
 $INCLUDE dictionary.cisco.asa
 #
@@ -167,133 +167,133 @@ $INCLUDE dictionary.cisco.asa
 #   Note : the altiga dictionary, not listed here, also uses the same Vendor ID
 #
 #$INCLUDE dictionary.cisco.vpn3000
-$INCLUDE dictionary.cisco.vpn5000
-$INCLUDE dictionary.cisco.bbsm
-$INCLUDE dictionary.citrix
-$INCLUDE dictionary.clavister
-$INCLUDE dictionary.cnergee
-$INCLUDE dictionary.colubris
-$INCLUDE dictionary.compatible
-$INCLUDE dictionary.cosine
-$INCLUDE dictionary.dante
-$INCLUDE dictionary.dellemc
-$INCLUDE dictionary.dlink
-$INCLUDE dictionary.digium
-$INCLUDE dictionary.dragonwave
-$INCLUDE dictionary.efficientip
-$INCLUDE dictionary.eltex
-$INCLUDE dictionary.epygi
-$INCLUDE dictionary.erx
-$INCLUDE dictionary.equallogic
-$INCLUDE dictionary.ericsson
-$INCLUDE dictionary.ericsson.ab
-$INCLUDE dictionary.ericsson.packet.core.networks
-$INCLUDE dictionary.extreme
-$INCLUDE dictionary.f5
-$INCLUDE dictionary.fdxtended
+#$INCLUDE dictionary.cisco.vpn5000
+#$INCLUDE dictionary.cisco.bbsm
+#$INCLUDE dictionary.citrix
+#$INCLUDE dictionary.clavister
+#$INCLUDE dictionary.cnergee
+#$INCLUDE dictionary.colubris
+#$INCLUDE dictionary.compatible
+#$INCLUDE dictionary.cosine
+#$INCLUDE dictionary.dante
+#$INCLUDE dictionary.dellemc
+#$INCLUDE dictionary.dlink
+#$INCLUDE dictionary.digium
+#$INCLUDE dictionary.dragonwave
+#$INCLUDE dictionary.efficientip
+#$INCLUDE dictionary.eltex
+#$INCLUDE dictionary.epygi
+#$INCLUDE dictionary.erx
+#$INCLUDE dictionary.equallogic
+#$INCLUDE dictionary.ericsson
+#$INCLUDE dictionary.ericsson.ab
+#$INCLUDE dictionary.ericsson.packet.core.networks
+#$INCLUDE dictionary.extreme
+#$INCLUDE dictionary.f5
+#$INCLUDE dictionary.fdxtended
 $INCLUDE dictionary.freeradius
-$INCLUDE dictionary.freeswitch
-$INCLUDE dictionary.fortinet
-$INCLUDE dictionary.foundry
-$INCLUDE dictionary.gandalf
-$INCLUDE dictionary.gemtek
-$INCLUDE dictionary.h3c
-$INCLUDE dictionary.hillstone
-$INCLUDE dictionary.hp
-$INCLUDE dictionary.huawei
-$INCLUDE dictionary.iea
-$INCLUDE dictionary.infonet
-$INCLUDE dictionary.issanni
-$INCLUDE dictionary.itk
-$INCLUDE dictionary.ipunplugged
-$INCLUDE dictionary.juniper
-$INCLUDE dictionary.karlnet
-$INCLUDE dictionary.kineto
-$INCLUDE dictionary.lancom
-$INCLUDE dictionary.lantronix
-$INCLUDE dictionary.livingston
-$INCLUDE dictionary.localweb
-$INCLUDE dictionary.lucent
-$INCLUDE dictionary.manzara
-$INCLUDE dictionary.meinberg
+#$INCLUDE dictionary.freeswitch
+#$INCLUDE dictionary.fortinet
+#$INCLUDE dictionary.foundry
+#$INCLUDE dictionary.gandalf
+#$INCLUDE dictionary.gemtek
+#$INCLUDE dictionary.h3c
+#$INCLUDE dictionary.hillstone
+#$INCLUDE dictionary.hp
+#$INCLUDE dictionary.huawei
+#$INCLUDE dictionary.iea
+#$INCLUDE dictionary.infonet
+#$INCLUDE dictionary.issanni
+#$INCLUDE dictionary.itk
+#$INCLUDE dictionary.ipunplugged
+#$INCLUDE dictionary.juniper
+#$INCLUDE dictionary.karlnet
+#$INCLUDE dictionary.kineto
+#$INCLUDE dictionary.lancom
+#$INCLUDE dictionary.lantronix
+#$INCLUDE dictionary.livingston
+#$INCLUDE dictionary.localweb
+#$INCLUDE dictionary.lucent
+#$INCLUDE dictionary.manzara
+#$INCLUDE dictionary.meinberg
 $INCLUDE dictionary.meraki
-$INCLUDE dictionary.merit
-$INCLUDE dictionary.meru
-$INCLUDE dictionary.microsemi
+#$INCLUDE dictionary.merit
+#$INCLUDE dictionary.meru
+#$INCLUDE dictionary.microsemi
 $INCLUDE dictionary.microsoft
-$INCLUDE dictionary.mikrotik
-$INCLUDE dictionary.mimosa
-$INCLUDE dictionary.motorola
-$INCLUDE dictionary.motorola.wimax
-$INCLUDE dictionary.navini
-$INCLUDE dictionary.netscreen
-$INCLUDE dictionary.networkphysics
-$INCLUDE dictionary.nexans
-$INCLUDE dictionary.ntua
-$INCLUDE dictionary.nokia
+#$INCLUDE dictionary.mikrotik
+#$INCLUDE dictionary.mimosa
+#$INCLUDE dictionary.motorola
+#$INCLUDE dictionary.motorola.wimax
+#$INCLUDE dictionary.navini
+#$INCLUDE dictionary.netscreen
+#$INCLUDE dictionary.networkphysics
+#$INCLUDE dictionary.nexans
+#$INCLUDE dictionary.ntua
+#$INCLUDE dictionary.nokia
 #
 #  Commented out because of attribute conflicts.
 #
 #$INCLUDE dictionary.nokia.conflict
-$INCLUDE dictionary.nomadix
-$INCLUDE dictionary.nortel
+#$INCLUDE dictionary.nomadix
+#$INCLUDE dictionary.nortel
 #
 #  Commented out because of attribute conflicts.
 #
 #$INCLUDE dictionary.openser
-$INCLUDE dictionary.packeteer
-$INCLUDE dictionary.paloalto
-$INCLUDE dictionary.patton
-$INCLUDE dictionary.perle
-$INCLUDE dictionary.propel
-$INCLUDE dictionary.prosoft
-$INCLUDE dictionary.proxim
-$INCLUDE dictionary.purewave
-$INCLUDE dictionary.quiconnect
-$INCLUDE dictionary.quintum
-$INCLUDE dictionary.redcreek
-$INCLUDE dictionary.riverbed
-$INCLUDE dictionary.riverstone
-$INCLUDE dictionary.roaringpenguin
-$INCLUDE dictionary.ruggedcom
-$INCLUDE dictionary.ruckus
-$INCLUDE dictionary.sangoma
-$INCLUDE dictionary.shasta
-$INCLUDE dictionary.sg
-$INCLUDE dictionary.shiva
-$INCLUDE dictionary.siemens
-$INCLUDE dictionary.slipstream
-$INCLUDE dictionary.sofaware
-$INCLUDE dictionary.softbank
-$INCLUDE dictionary.sonicwall
-$INCLUDE dictionary.springtide
-$INCLUDE dictionary.starent
-$INCLUDE dictionary.surfnet
-$INCLUDE dictionary.symbol
-$INCLUDE dictionary.telebit
-$INCLUDE dictionary.terena
-$INCLUDE dictionary.trapeze
-$INCLUDE dictionary.travelping
-$INCLUDE dictionary.tropos
-$INCLUDE dictionary.t_systems_nova
-$INCLUDE dictionary.ukerna
-$INCLUDE dictionary.unix
-$INCLUDE dictionary.usr
-$INCLUDE dictionary.utstarcom
-$INCLUDE dictionary.valemount
-$INCLUDE dictionary.versanet
-$INCLUDE dictionary.waverider
-$INCLUDE dictionary.walabi
-$INCLUDE dictionary.wichorus
-$INCLUDE dictionary.wifialliance
-$INCLUDE dictionary.wimax
+#$INCLUDE dictionary.packeteer
+#$INCLUDE dictionary.paloalto
+#$INCLUDE dictionary.patton
+#$INCLUDE dictionary.perle
+#$INCLUDE dictionary.propel
+#$INCLUDE dictionary.prosoft
+#$INCLUDE dictionary.proxim
+#$INCLUDE dictionary.purewave
+#$INCLUDE dictionary.quiconnect
+#$INCLUDE dictionary.quintum
+#$INCLUDE dictionary.redcreek
+#$INCLUDE dictionary.riverbed
+#$INCLUDE dictionary.riverstone
+#$INCLUDE dictionary.roaringpenguin
+#$INCLUDE dictionary.ruggedcom
+#$INCLUDE dictionary.ruckus
+#$INCLUDE dictionary.sangoma
+#$INCLUDE dictionary.shasta
+#$INCLUDE dictionary.sg
+#$INCLUDE dictionary.shiva
+#$INCLUDE dictionary.siemens
+#$INCLUDE dictionary.slipstream
+#$INCLUDE dictionary.sofaware
+#$INCLUDE dictionary.softbank
+#$INCLUDE dictionary.sonicwall
+#$INCLUDE dictionary.springtide
+#$INCLUDE dictionary.starent
+#$INCLUDE dictionary.surfnet
+#$INCLUDE dictionary.symbol
+#$INCLUDE dictionary.telebit
+#$INCLUDE dictionary.terena
+#$INCLUDE dictionary.trapeze
+#$INCLUDE dictionary.travelping
+#$INCLUDE dictionary.tropos
+#$INCLUDE dictionary.t_systems_nova
+#$INCLUDE dictionary.ukerna
+#$INCLUDE dictionary.unix
+#$INCLUDE dictionary.usr
+#$INCLUDE dictionary.utstarcom
+#$INCLUDE dictionary.valemount
+#$INCLUDE dictionary.versanet
+#$INCLUDE dictionary.waverider
+#$INCLUDE dictionary.walabi
+#$INCLUDE dictionary.wichorus
+#$INCLUDE dictionary.wifialliance
+#$INCLUDE dictionary.wimax
 $INCLUDE dictionary.wispr
-$INCLUDE dictionary.xedia
-$INCLUDE dictionary.xylan
-$INCLUDE dictionary.yubico
-$INCLUDE dictionary.zeus
-$INCLUDE dictionary.zte
-$INCLUDE dictionary.zyxel
+#$INCLUDE dictionary.xedia
+#$INCLUDE dictionary.xylan
+#$INCLUDE dictionary.yubico
+#$INCLUDE dictionary.zeus
+#$INCLUDE dictionary.zte
+#$INCLUDE dictionary.zyxel
 
 #
 #	And finally the server internal attributes.
-- 
2.18.0

